### PR TITLE
Update validate_zfcp_multipath to current infrastructure

### PR DESCRIPTION
**Pull request by manfredi**
- Description: Update validate_zfcp_multipath to current infrastructure
- Related ticket: [poo#152053](https://progress.opensuse.org/issues/152053)
- Verification run: [overview](https://openqa.suse.de/tests/overview?build=issues-152053)

### CODE REVIEW

File: tests/console/validate_zfcp_multipath.pm [hunk 1]
```diff
@@ -33,6 +33,12 @@ sub verify_plain_scsi_block_devices {
     assert_equals 4, $small_disks, "Expected 4 SCSI block devices sized 42.9GB at LUN 0x4001405000000000, found $small_disks";
 }
 
+sub verify_plain_scsi_block_devices_2_block {
```
Comment (lemon-suse) [inline tests/console/validate_zfcp_multipath.pm:36]: I think you can replace the old function directly and update the description for the function for your change.

File: tests/console/validate_zfcp_multipath.pm [hunk 2]
```diff
@@ -50,11 +56,23 @@ sub verify_multipath_block_devices {
...
+    my $disks = scalar grep { /\bsd(a|b)\b/ } @mpath;
+    assert_equals 2, $disks, "50GB block device does not have 2 paths";
+}
+
 sub run {
     select_console 'root-console';
     verify_host_bus_adapters_attached("0.0.fa00", "0.0.fc00");
-    verify_plain_scsi_block_devices;
-    verify_multipath_block_devices;
+    # verify_plain_scsi_block_devices;
```
Comment (lemon-suse) [inline tests/console/validate_zfcp_multipath.pm:72]: Do not need change here if we just update the old function.

File: tests/console/validate_zfcp_multipath.pm (hunk 1 above)
Comment (manfredi) [reply tests/console/validate_zfcp_multipath.pm]: Done

File: tests/console/validate_zfcp_multipath.pm (hunk 2 above)
Comment (manfredi) [reply tests/console/validate_zfcp_multipath.pm]: Done

File: tests/console/validate_zfcp_multipath.pm [hunk 3]
```diff
@@ -23,31 +23,20 @@ sub verify_host_bus_adapters_attached {
     }
 }
 
-# Verify that there are 8 SCSI block devices listed (as a result of having two adapters, two hard disks and two paths to each disk, 2^3 = 8)
-# Verify that LUN 0x4001403200000000 corresponds to the 214GB disk, and LUN 0x4001405000000000 to the 42.9GB disk
```
Comment (lemon-suse) [inline tests/console/validate_zfcp_multipath.pm:27]: # Verify that there are 2 SCSI block devices listed
# Verify that LUN 0x0000000000000000 corresponds to 53.6GB disk

Update the description like this?

Comment (manfredi) [reply tests/console/validate_zfcp_multipath.pm:27]: Updated comments

**Review by lemon-suse — APPROVED** [review]

**Review by JRivrain — APPROVED** [review]

**Pull request merged**